product_creation_prevention/__manifest__.py

```diff
@@ -1,9 +1,8 @@
 {
     "name": "Product Creation Prevention",
     "version": "15.0.1.0.0",
-    "summary": """Product Creation Prevention
-    By adding access right we are preventing users to create products 'on the fly'
-    """,
+    "summary": """Using this module we can prevent to create new products by
+    users except admin user""",
```

Comment:
I understood that product creation prevention now also applies to admin users. Doesn't it?